pkg/services/accesscontrol/seeder/roles.go

```diff
@@ -0,0 +1,88 @@
+package seeder
+
+import "github.com/grafana/grafana/pkg/services/accesscontrol"
+
+var builtInRoles = []accesscontrol.RoleDTO{
+	{
+		Name:    "grafana:builtin:users:read:self",
+		Version: 1,
+		Permissions: []accesscontrol.Permission{
+			{
+				Permission: "users:read",
+				Scope:      accesscontrol.ScopeUsersSelf,
+			},
+			{
+				Permission: "users.tokens:list",
+				Scope:      accesscontrol.ScopeUsersSelf,
+			},
+			{
+				Permission: "users.teams:read",
+				Scope:      accesscontrol.ScopeUsersSelf,
+			},
+		},
+	},
+	{
+		Name:    "roles:adminUsers:viewer",
```

Comment:
We should have a prefix for all of our predefined roles that we can instruct/prevent users from creating via the APIs.
I'd also suggest making all "manage users" roles sort under `users`, regardless of if the actions are intended for administration or end-users. We're probably going to end up sorting these alphabetically when presenting to users, so I think it's rather important that the alphabetical ordering groups related concepts near each other.
Finally, I suggest that we call it "view" rather than "viewer", or something else like "read" or "list". Naming is hard, but keeping words to their base forms seems reasonable to me. 

The prefix thing seems like a mistake not to do, everything else I'll yield on any amount of resistance, it's my reasoning behind how I decided to name the above `grafana:builtin:users:read:self` (which, for the record, isn't a very good name either (the prefix is too long, so it risks being the only thing users see in a list), so I wouldn't use it as a template).

With those modifications we'd end up with something like the following:

```
gf:users:self:view
gf:users:admin:view
gf:users:admin:edit
```

Reply:
I agree with reasoning 👍 I'd also vote for `read` instead of `view` , since it's more known as a term in the IAM world + view is harder to reason about. 

Also the prefix makes sense, have not thought that we would need to apply some prevention logic for these roles. Since it's a convention we want to stick, I would say `gf` prefix works just fine and will save us few bytes of space in the storage.

Reply:
`read` makes more sense to me. `list` fits well for collections, but I'm not sure we want to differentiate `list` (when user allowed to list resources, but not read it, like you can list users, but cannot read user profile) from `read`.

Reply:
In our case users search returns items with lots of info about users, so `read` makes more sense than `list` I think.

Reply:
Agreed, list makes sense when there's a value to differentiating accessing something and knowing _that_ it exists. Generally speaking, I think we should avoid that to keep things simple 🙂